src/app/[locale]/_component/accordion/faqDefault.tsx

```diff
@@ -2,52 +2,107 @@ import React from 'react';
 import { Grid, Typography } from '@mui/material';
 import { AccordionCustom } from './accordion';
 
-export function FAQDefault() {
+interface FAQProps {
+  flow?: string;
+}
+
+interface FAQEntries {
+  summary: string;
+  details: string;
+}
+
+export function FAQ({ flow = 'DEFAULT' }: FAQProps) {
```

Comment:
Why we are using a mix of normal and arrow function to define components? Pick a style and be consistent.

Reply:
Changed the component to an arrow Function.